dist/goldenlayout.min.js

```diff
@@ -1,3 +0,0 @@
".lm_tabdropdown_list"),this.tabDropdownContainer.hide(),this.controlsContainer=this.element.find(".lm_controls"),this.parent=n,this.parent.on("resize",this._updateTabSizes,this),this.tabs=[],this.activeContentItem=null,this.closeButton=null,this.tabDropdownButton=null,this.hideAdditionalTabsDropdown=e.utils.fnBind(this._hideAdditionalTabsDropdown,this),t(document).mouseup(this.hideAdditionalTabsDropdown),this._lastVisibleTabIndex=-1,this._tabControlOffset=this.layoutManager.config.settings.tabControlOffset,this._createControls()},e.controls.Header._template=['<div class="lm_header">','<ul class="lm_tabs"></ul>','<ul class="lm_controls"></ul>','<ul class="lm_tabdropdown_list"></ul>',"</div>"].join(""),e.utils.copy(e.controls.Header.prototype,{createTab:function(t,i){var n,s;for(s=0;s<this.tabs.length;s++)if(this.tabs[s].contentItem===t)return;return n=new e.controls.Tab(this,t),0===this.tabs.length?(this.tabs.push(n),void this.tabsContainer.append(n.element)):(void 0===i&&(i=this.tabs.length),i>0?this.tabs[i-1].element.after(n.element):this.tabs[0].element.before(n.element),this.tabs.splice(i,0,n),void this._updateTabSizes())},removeTab:function(t){for(var e=0;e<this.tabs.length;e++)if(this.tabs[e].contentItem===t)return this.tabs[e]._$destroy(),void this.tabs.splice(e,1);throw new Error("contentItem is not controlled by this header")},setActiveContentItem:function(t){var e,i,n,s;for(e=0;e<this.tabs.length;e++)n=this.tabs[e].contentItem===t,this.tabs[e].setActive(n),n===!0&&(this.activeContentItem=t,this.parent.config.activeItemIndex=e);if(this.layoutManager.config.settings.reorderOnTabMenuClick&&this._lastVisibleTabIndex!==-1&&this.parent.config.activeItemIndex>this._lastVisibleTabIndex){for(s=this.tabs[this.parent.config.activeItemIndex],i=this.parent.config.activeItemIndex;i>0;i--)this.tabs[i]=this.tabs[i-1];this.tabs[0]=s,this.parent.config.activeItemIndex=0}this._updateTabSizes(),this.parent.emitBubblingEvent("stateChanged")},position:function(t){var e=this.parent._header.show;return e&&!this.parent._side&&(e="top"),void 0!==t&&this.parent._header.show!=t&&(this.parent._header.show=t,this.parent._setupHeaderPosition()),e},_$setClosable:function(t){return!(!this.closeButton||!this._isClosable())&&(this.closeButton.element[t?"show":"hide"](),!0)},_$destroy:function(){this.emit("destroy",this);for(var e=0;e<this.tabs.length;e++)this.tabs[e]._$destroy();t(document).off("mouseup",this.hideAdditionalTabsDropdown),this.element.remove()},_getHeaderSetting:function(t){if(t in this.parent._header)return this.parent._header[t]},_createControls:function(){var t,i,n,s,o,r,a,h,l;l=e.utils.fnBind(this._showAdditionalTabsDropdown,this),h=this.layoutManager.config.labels.tabDropdown,this.tabDropdownButton=new e.controls.HeaderButton(this,h,"lm_tabdropdown",l),this.tabDropdownButton.element.hide(),this._getHeaderSetting("popout")&&(i=e.utils.fnBind(this._onPopoutClick,this),n=this._getHeaderSetting("popout"),new e.controls.HeaderButton(this,n,"lm_popout",i)),
```

Comment:
Should these files be in source control at all? In modern codebases, they rarely are.. I think bower is the main reason they must be..

Reply:
I never looked into bower. Don't know, sorry.

Reply:
Should be ok either way. Its not needed but its also nice to have the result of a build in the repo

Reply:
IMHO it's really bad to have files like such in the repo, as they change a lot, and will have tons of merge conflicts.

Basically they are are artifacts, not code.